shared/constants/countries.ts

```diff
@@ -0,0 +1,259 @@
+enum Country {
```

Comment:
nit: prefer export as non-default:

` export enum`

Reply:
Addressed in https://github.com/opengovsg/FormSG/pull/3705